dbms/src/Common/TiFlashMetrics.h

```diff
@@ -232,6 +232,9 @@ namespace DB
         F(type_merged_task, {{"type", "merged_task"}}, ExpBuckets{0.001, 2, 20}))                                                         \
     M(tiflash_mpp_task_manager, "The gauge of mpp task manager", Gauge,                                                                   \
         F(type_mpp_query_count, {"type", "mpp_query_count"}))                                                                             \
+    M(tiflash_storage_read_duration_seconds, "Bucketed histogram read duration", Histogram,                                               \
+        F(type_table_scan_shared, {{"type", "table_scan_shared"}}, ExpBuckets{0.001, 2, 20}),                                             \
+        F(type_table_scan_raw, {{"type", "table_scan_raw"}}, ExpBuckets{0.001, 2, 20}))                                                   \
```

Comment:
`type_table_scan_fast` and `type_table_scan` maybe more suitable?
`type_table_scan_fast` responding to `FastScan` in the doc https://docs.pingcap.com/tidb/dev/dev-guide-use-fastscan